pkg/names/names.go

```diff
@@ -119,10 +119,13 @@ var (
 		{"Nat", "NAT", "nat", regexp.MustCompile("Nat(?!i)", regexp.None)},
 		{"Oidc", "OIDC", "oidc", nil},
 		{"Ocsp", "OCSP", "ocsp", nil},
+		// Model fields starting with 'Ram' refer to RAM
+		{"Ram", "RAM", "ram", regexp.MustCompile("Ram", regexp.None)},
```

Comment:
The rule for `RAM` is not exhaustive, ex: rambunctious, ramification, Ramadan. However, I deep dived the models to validate this simpler rule should work for all controllers.

In **aws-sdk-go/models/apis** , executed `grep -r '[a-zA-Z]*ram[a-zA-Z]*' -o . | sort | uniq`:

```
addThingsToThingGroupParams
algorithmHyperParameters
cacheKeyParameters
categoricalHyperParameterRanges
containerParams
continuousHyperParameterRanges
datagram
datagrams
diagram
dropframe
emitProgram
enableIoTLoggingParams
exactFramerate
frame
frameMetric
frameMetricData
frameMetrics
frameName
framerate
framerateControl
framerateDenominator
framerateNumerator
framerates
frames
framework
frameworkId
frameworkMetadataList
frameworkName
frameworkType
frameworks
gopNumBFrames
grammar
histogram
hyperparameter
hyperparameters
iframe
integerHyperParameterRanges
keyframe
linuxProcessParams
logParameters
maxFrameAverageLightLevel
mydbclusterparametergroup
mydbparametergroup
mydbparamgroup
mymysqlparametergroup
numberBFrames
numberBFramesBetweenReferenceFrames
param
parameter
parameterName
parameterValue
parameterized
parameters
paramount
params
paramstore
program
programDateTimeIntervalSeconds
programName
programNumber
programlisting
programmatic
programmatically
programmer
programming
programs
publishFindingToSnsParams
ram
ramdisk
replaceDefaultPolicyVersionParams
requestParameterKey
requestParameters
responseParameters
scram
signingProfileParameter
timeframe
updateCACertificateParams
updateDeviceCertificateParams
```